tests/storage/storage.go

```diff
@@ -646,21 +649,21 @@ var _ = SIGDescribe("Storage", func() {
 
 			// Not a candidate for testing on NFS because the VMI is restarted and NFS PVC can't be re-used
 			It("[test_id:3138]should start vmi multiple times", func() {
-				vmi = tests.NewRandomVMIWithPVC(tests.DiskAlpineHostPath)
-				tests.AddPVCDisk(vmi, "disk1", v1.DiskBusVirtio, tests.DiskCustomHostPath)
+				vmiSpec := libvmi.NewAlpine(libvmi.WithPersistentVolumeClaim(tests.DiskAlpineHostPath))
```

Comment:
nit: Why did you changed the name of the variable `vmi`?

Reply:
You are right, it is not consistent with the remaining tests.

For me it is much easier to differentiate the spec of vm that we define from the actual VM we get from k8s, by using different variables. And it is easier to spot a problem where someone expects to get actual vmi, but has 
`_, err := client.Get` in code, so is still using the original spec.

But it is not needed here. We can vote which way is better. Anyway i can remove it to make this PR contain only changes related to libvmi changes. The rest can be included in https://github.com/kubevirt/kubevirt/pull/7634.

Reply:
For me it just raised questions. I will leave it to you and the maintainer to decide what is best.
Generally speaking, such changes cause the reviewer to look for the catch.